res/xml/preferences_game.xml

```diff
@@ -3,108 +3,86 @@
     xmlns:mupen64="http://schemas.android.com/apk/res/org.mupen64plusae.v3.alpha"
     android:key="screenPlayMenuActivity" >
 
-    <Preference
-        android:icon="@drawable/ic_play"
-        android:key="actionResume"
-        android:summary="@string/actionResume_summary"
-        android:title="@string/actionResume_title" />
-    <Preference
-        android:icon="@drawable/ic_undo"
-        android:key="actionRestart"
-        android:summary="@string/actionRestart_summary"
-        android:title="@string/actionRestart_title" />
-    <Preference
-        android:icon="@drawable/ic_settings"
-        android:key="screenGlobalSettings"
-        android:summary="@string/screenGlobalSettings_summary"
-        android:title="@string/screenGlobalSettings_title" >
-        <intent android:action=".SettingsGlobalActivity" />
-    </Preference>
-
-    <PreferenceCategory
```

Comment:
New layout is identical, except without being wrapped inside the PreferenceCategory. This means everything had to be re-indented by 4 spaces.